pom.xml

```diff
@@ -54,6 +54,13 @@
         <type>pom</type>
         <scope>import</scope>
       </dependency>
+      <dependency>
+        <groupId>software.amazon.awssdk</groupId>
+        <artifactId>bom</artifactId>
+        <version>2.25.67</version>
```

Comment:
Is there specific reason to use this version, instead of later/latest (https://mvnrepository.com/artifact/software.amazon.awssdk/bom), 2.25.70 / 2.26.12 ?

Reply:
Just copied it from the bedrock examples. Will change it to newer version.